src/components/pages/personal-teacher-page/personal-teacher-tabs/components/general-tab/GeneralTab.styles.ts

```diff
@@ -0,0 +1,105 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const wrapper: SxProps<Theme> = {
+  marginTop: '-15px',
```

Comment:
how that happened that we need a negative margin?

Reply:
life....